packages/components/src/components/tag/tag.tsx

```diff
@@ -8,14 +8,22 @@ import classNames from "classnames";
   shadow: true
 })
 export class Tag {
-  /** (optional) Button variant */
+  /** (optional) Tag variant */
   @Prop() variant?: string = "";
+  /** (optional) Tag on an <a> element */
+  @Prop() link?: boolean = false;
 
   private getCssClassMap(): CssClassMap {
     return classNames("tag", this.variant && `tag--variant-${this.variant}`);
   }
 
   render() {
+    if (this.link)
+      return (
+        <a href="#" class={this.getCssClassMap()}>
```

Comment:
... href="#" ... would return the 'this.link' prop

Reply:
the code is outdated. can you check updated version